hail/python/test/hail/table/test_table.py

```diff
@@ -599,7 +599,7 @@ def test_localize_entries(self):
         ref_tab = hl.Table.parallelize(ref_data, ref_schema).key_by('row_idx')
         mt = hl.utils.range_matrix_table(8, 6)
         mt = mt.annotate_entries(v=mt.row_idx+mt.col_idx)
-        t = mt._localize_entries('__entries')
+        t = mt._localize_entries('__entries', '__cols').drop('__cols')
```

Comment:
might be nice to expand this test to include col field preservation instead of dropping the cols.